dotCMS/src/main/java/com/dotmarketing/velocity/directive/DotParse.java

```diff
@@ -117,7 +122,9 @@ String resolveTemplatePath(final Context context, final Writer writer, final Ren
       if( e instanceof ResourceNotFoundException ) {
         throw (ResourceNotFoundException) e;
       }
-      
+      else if(e instanceof DotSecurityException){
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace the usage of the "instanceof" operator by a catch block. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1193)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this "else" on the same line that the previous closing curly brace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)